src/librustdoc/html/render/span_map.rs

```diff
@@ -63,32 +64,59 @@ struct SpanMapVisitor<'tcx> {
 
 impl<'tcx> SpanMapVisitor<'tcx> {
     /// This function is where we handle `hir::Path` elements and add them into the "span map".
-    fn handle_path(&mut self, path: &rustc_hir::Path<'_>, path_span: Option<Span>) {
+    fn handle_path(&mut self, path: &rustc_hir::Path<'_>) {
```

Comment:
Why does handle_path not need path_span anymore?

Reply:
The last argument was always `None` so this is basically a cleanup that was forgotten in a previous update.